doc/cabal-project-description-file.rst

```diff
@@ -253,9 +266,10 @@ There is no command line variant of this stanza.
 
 .. _source-repository-package-fields:
 
-The :ref:`VCS fields<vcs-fields>` of ``source-repository-package`` are:
+The fields of ``source-repository-package`` are:
 
 ..
+  -- Cabal internal datatype
```

Comment:
Likewise, I don't think the source code helps here.

Reply:
Removed